lib/apiservers/engine/backends/archive.go

```diff
@@ -644,3 +687,90 @@ func (rm *ArchiveStreamReaderMap) Close() {
 
 	rm.prefixTrie.Visit(closeStream)
 }
+
+// try to fake the statpath for path that targets the mountpoint or along the mountpoint
```

Comment:
`// tryFakeStatPath tries to fake the statpath for path that targets the mountpoint or along the mountpoint`